### PR TITLE
refactor(core): delegate the otrs configuration to the host app

### PR DESCRIPTION
**Pull request by antleblanc**
# Delegate configuration

### :nail_care: Refactor

2dccfa9 - refactor(core): delegate the otrs configuration to the host app
f1c3c3e - refactor(apps.public.cloud): move components into one module

### :house: Internal

- No QC required.

### :boom: Breaking Change

- OTRS configuration no more rely on core module

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1599,6 +1599,14 @@
     rollup-pluginutils "^2.3.3"
     slash "^2.0.0"
 
+"@ovh-ux/manager-navbar@^0.2.0-alpha.2":
```
Comment (antleblanc) [inline yarn.lock]: `yarn install` from `decfde28ade6581f031b7d618c5a2f5c3782a995` causes this following change.

Comment (jleveugle) [reply yarn.lock]: fixed on `develop` branch

Comment (antleblanc) [reply yarn.lock]: branch `bugfix/module-core` is now rebase on `develop`.

**Comment by antleblanc** [comment]
Rebased on `7ef3b3fbeeb7e42051f31c6ecea6f59d57d3aa30` (since #640 was merged).

**Comment by jleveugle** [comment]
Hello @antleblanc,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**